test/Chainweb/Test/SPV.hs

```diff
@@ -359,49 +358,48 @@ apiTests rdb tls v = withTestPayloadResource rdb v 100 (\_ _ -> return ()) $ \db
 
 txApiTests :: PayloadCasLookup cas => IO (TestClientEnv_ cas) -> Step -> IO ()
 txApiTests envIO step = do
-    PayloadTestClientEnv env cutDb payloadDbs v <- envIO
-    give (snd . head $ payloadDbs) $ do
```

Comment:
Removing this line is the only real change here. The rest is just reindenting the function body.